notification/src/vm2SandboxExecutor.ts

```diff
@@ -0,0 +1,38 @@
+import { VM } from 'vm2'
+
+import SandboxExecutor from './interfaces/sandboxExecutor';
+
+
+const FUNCTION_WRAP_PREFIX_LENGTH = 1
+
+export default class VM2SandboxExecutor implements SandboxExecutor {
+  vm: VM;
+
+  constructor (timeout = 5000) {
+    this.vm = new VM({
+      timeout
+    })
+  }
+
+  evaluate (expression: string, data: object): boolean {
+    const wrapper =
+      'f=function(data){' +
+      'return ' +
+      expression +
+      '};f(' +
+      JSON.stringify(data) +
+      ');'
+
+    let result = false
+    try {
+      result = this.vm.run(wrapper)
+    } catch (err) {
+      // console.error('Malformed expression received: ' + expression, err)
+    }
+    if (typeof result !== 'boolean') {
+      return false
```

Comment:
maybe we should log something here, too

Reply:
Throw error instead, so there is log output